### DISCUSSION TITLE
Map not showing when saving as html

### DISCUSSION THREAD

**Discussion by Devvver** [opengeos/leafmap#399] [Q&A] [answered]
I use the simplest code
`import leafmap
m = leafmap.Map(center=(40, -100), zoom=4)
m.save("map.html")`

Execute, get html file
[https://skr.sh/i/240323/Ubt22jFD.jpg?download=1&name=%D0%A1%D0%BA%D1%80%D0%B8%D0%BD%D1%88%D0%BE%D1%82%2024-03-2023%2018:19:20.jpg](url)
bugs in browser development environment
[https://skr.sh/i/240323/AAUQbzRp.jpg?download=1&name=%D0%A1%D0%BA%D1%80%D0%B8%D0%BD%D1%88%D0%BE%D1%82%2024-03-2023%2018:20:15.jpg](url)

The library does not support work in the browser, but only in jupyter notebook?
Or am I doing something wrong?

**Comment by giswqs** [comment, marked as answer]
ipyleaflet has some issues with saving html. Try `import leafmap.foliumap as leafmap` rather than `import leafmap`
